src/util/PopulationsMap.ts

```diff
@@ -1,5 +1,7 @@
 import { MeasureScoring } from "../models/MeasureScoring";
 import { MeasurePopulation } from "../models/MeasurePopulation";
+import { Group } from "../models/Measure";
```

Comment:
LGTM is complaining that this GroupPopulation 🤷  imports isn't used.

Reply:
Yeah, I removed them Thanks greg